pastefile/app.py

```diff
@@ -17,7 +17,14 @@
 formatter_stream = logging.Formatter('%(message)s')
 hdl_stream.setFormatter(formatter_stream)
 LOG.addHandler(hdl_stream)
-
+app.config.setdefault('UPLOAD_FOLDER', "/opt/pastefile/files/")
```

Comment:
It could be better to add this in a function like `init_default_config` or at least Add a new line and a comment.

Don't let this block stick to the logging part, it's not the same thing